scripts/un/sdg/cities.csv

```diff
@@ -0,0 +1,1774 @@
+SAN_JUAN_SACATEPEQUEZ,wikidataId/Q127622
+QUETZALTENANGO,wikidataId/Q334577
+PUERTO_BARRIOS,wikidataId/Q991400
+MAZATENANGO,wikidataId/Q1023983
+GUATEMALA_CITY,wikidataId/Q1555
+ESCUINTLA,wikidataId/Q780771
+COBAN,wikidataId/Q867077
+SAINTMARC,wikidataId/Q132189
+PORT-AU-PRINCE,wikidataId/Q34261
+CAP_HAITIEN,wikidataId/Q216835
+EL_PROGRESO,wikidataId/Q2277161
+COMAYAGUA,wikidataId/Q679169
+TEGUCIGALPA,wikidataId/Q3238
+SAN_PEDRO_SULA,wikidataId/Q274584
+LA_CEIBA,wikidataId/Q916536
+VESZPREM,wikidataId/Q146317
+RAKOSLIGET,wikidataId/Q714696
+PECS,wikidataId/Q45779
+TATABANYA,wikidataId/Q187821
+SZOMBATHELY,wikidataId/Q42007
+SZEKESFEHERVAR,wikidataId/Q130212
+SZEGED,wikidataId/Q81581
+NYIREGYHAZA,wikidataId/Q171223
+MISKOLC,wikidataId/Q102397
+KECSKEMET,wikidataId/Q171357
+GYOR,wikidataId/Q134494
+ERD,wikidataId/Q193050
+DEBRECEN,wikidataId/Q79880
+BUDAPEST,nuts/HU101
+REYKJAVIK,wikidataId/Q1764
+PARBHANI,wikidataId/Q1797389
+SURAT,wikidataId/Q1797317
+SITAPUR,wikidataId/Q1812539
+SINGRAULI,wikidataId/Q2668638
+PUNE,wikidataId/Q1538
+PATNA,wikidataId/Q100077
+PALI,wikidataId/Q46925
+NELLORE,wikidataId/Q61434
+MUMBAI,wikidataId/Q1156
+MORENA,wikidataId/Q2341467
+MALEGAON,wikidataId/Q580965
+KOZHIKODE,wikidataId/Q1142979
+KOLKATA,wikidataId/Q1348
+KANPUR,wikidataId/Q66568
+KANCHIPURAM,wikidataId/Q15157
+JALNA,wikidataId/Q1804863
+JAIPUR,wikidataId/Q1134781
+BELGAUM,wikidataId/Q270176
+HINDUPUR,wikidataId/Q760056
+COIMBATORE,wikidataId/Q15136
+CHANDIGARH,wikidataId/Q43433
+BHIWANDI,wikidataId/Q645725
+BENGALURU,wikidataId/Q1355
+BAHARAMPUR,wikidataId/Q633530
+AHMEDABAD,wikidataId/Q1070
+THOOTHUKKUDI,wikidataId/Q41562
+HYDERABAD_INDIA,wikidataId/Q1361
+VIJAYAWADA,wikidataId/Q200017
+TUMKUR,wikidataId/Q911310
+PEKALONGAN,wikidataId/Q10623
+BANDA_ACEH,wikidataId/Q5779
+BANDUNG,wikidataId/Q10332
+CIREBON,wikidataId/Q10368
+SURABAYA,wikidataId/Q11462
+SEMARANG,wikidataId/Q10687
+SAMARINDA,wikidataId/Q14499
+PURWAKARTA,wikidataId/Q10378
+PEMATANGSIANTAR,wikidataId/Q5979
+PEMALANG,wikidataId/Q10624
+JOMBANG,wikidataId/Q11082
+PAREPARE,wikidataId/Q14636
+PALEMBANG,wikidataId/Q8131
+MEDAN,wikidataId/Q5972
+MAKASSAR,wikidataId/Q14634
+KENDARI,wikidataId/Q15379
+JAMBI,wikidataId/Q2051
+GARUT_KOTA,wikidataId/Q833632
+BENGKULU,wikidataId/Q1890
+MASHAD,wikidataId/Q121157
+NISHABUR,wikidataId/Q131350
+PIRANSHAHR,wikidataId/Q3015863
+QOM,wikidataId/Q170573
+KHORAM_ABAD,wikidataId/Q502212
+KERMAN,wikidataId/Q171714
+KASHMAR,wikidataId/Q720403
+BANDAR_ABBAS,wikidataId/Q154814
+KARAJ,
+GORGAN,wikidataId/Q188155
+ESFAHAN,wikidataId/Q42053
+DEHDASHT,wikidataId/Q888365
+BOJNURD,wikidataId/Q317946
+BABOL,wikidataId/Q605157
+ARAK,wikidataId/Q212628
+ARADABIL,wikidataId/Q12727756
+AHVAZ,wikidataId/Q170366
+VARAMIN,wikidataId/Q518985
+TEHRAN,wikidataId/Q3616
+TABRIZ,wikidataId/Q80053
+SHIRAZ,wikidataId/Q6397066
+SHAHIN_SHAHR,wikidataId/Q862105
+SARI,wikidataId/Q168843
+SANANDAJ,wikidataId/Q272093
+SALMAS,wikidataId/Q765806
+KASAN,

```

Comment:
How many cities don't have mappings?  From spot checks what are reasons?

Reply:
this current list is missing 120/1773. from a quick look, some of these are regions not cities, but others are pretty major cities and we definitely have dcids for them. there might have just been some flakiness in api, but these could easily be added manually, or I can try to update the script to be more robust. I'm more worried about incorrect matches, but can follow up with some sanity validation based on lat/lon since un provided these